h/notification/reply.py

```diff
@@ -28,7 +28,9 @@ class Notification(
     """
 
 
-def get_notification(request, annotation, action):
+def get_notification(
```

Comment:
I think this is better than any low effort refactor. It has a lot of them because is doing early returns which looks fine to me.